mwaa/bucket/dags/modules/constants.py

```
@@ -0,0 +1,70 @@
+    try:
+        with open(f"{CONFIG_PATH}/{filename}", "r") as f:
+            return json.load(f)
+    except Exception as e:
+        if filename == "car.json":
+            return [
+                {"car_id": "santafe", "keywords": ["싼타페", "산타페"]},
+                {"car_id": "avante", "keywords": ["아반떼", "애반떼"]},
+            ]
+        if filename == "community.json":
+            return ["bobaedream", "clien"]
+        raise Exception(f"Failed to load config file {filename}: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)